common/shlibs

```diff
@@ -3400,3 +3400,6 @@ libsnoresettings-qt5.so.0.7 snorenotify-0.7.0_1
 libTECkit.so.0 libteckit-2.5.8_1
 libTECkit_Compiler.so.0 libteckit-2.5.8_1
 libwf-config.so wf-config-0.1_1
+libapt-private.so.0.0 apt-1.2.29_1
```

Comment:
please don't add unless using

Reply:
All binaries provided by this package link against this library. Also if not specified xbps-src complains when building the package:
> => WARNING: apt-1.7.2_1: libapt-private.so.0.0 not found in common/shlibs!                                                                                                                                                                                     
> => WARNING: apt-1.7.2_1: libapt-inst.so.2.0 not found in common/shlibs!                                                                                                                                                                                        
> => WARNING: apt-1.7.2_1: libapt-pkg.so.5.0 not found in common/shlibs!

Reply:
please don't add unless using